README.md

```diff
@@ -12,6 +12,16 @@ To learn more about why we created and maintain this boilerplate project, read o
 
 ## Usage
 
+### Package Manager
+
+Make sure that `yarn` is correctly configured before using the boilerplate. Otherwise, you might encounter some packaging errors when using other tools like `npm`.
+
+This also means that every command you would usualy run with `npm` needs to be run with `yarn` instead.
```

Comment:
```suggestion
This also means that every command you would usually run with `npm` needs to be run with `yarn` instead.
```

Reply:
We can still specify this for "power users" but I think we should talk more about the Makefile than Yarn here. You're not supposed to have to run a single yarn command since the Makefile contains all you need